daemon/k8s_watcher.go

```diff
@@ -234,6 +237,7 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 	serSvcs := serializer.NewFunctionQueue(20)
 	serEps := serializer.NewFunctionQueue(20)
 	serCNPs := serializer.NewFunctionQueue(20)
+	serPods := serializer.NewFunctionQueue(20)
```

Comment:
I think the queue must be much bigger for pods. I'd do at least 1024 so we are less likely to block.